src/Board.cpp

```diff
@@ -148,7 +148,78 @@ Lines Board::getCluesRows() const
     return cluesRows_;
 }
 
+bool Board::isSolved()
+{
+    return (not checkIfAnyFieldIsUnknown()) &&
+                verifyRows() &&
+                verifyCols();
+}
+
 void Board::display() const
 {
     std::cout << DisplayBoard::display(*this);
 }
+
+bool Board::checkIfAnyFieldIsUnknown()
+{
+    for(auto line : board_)
+        if (std::any_of(line.cbegin(),
```

Comment:
What do you think about this formatting
```cpp
        return std::any_of(line.cbegin(), line.cend(), [](const auto & elem)
        {
            return elem == BoardFields::UNKNOWN;
        });
```
and a reference in lamba ;)
BTW it looks that you can return result of any_of, without having if

Reply:
It still have to iterate through all lines of board_.
Best i can do is this:
```
    return std::all_of(board_.begin(), board_.end(), [](const auto line)
    {
        return std::any_of(line.cbegin(),  line.cend(), [](const auto elem)
        {
            return elem == BoardFields::UNKNOWN;
        });
    });
```